src/main/java/com/parasoft/findings/jenkins/coverage/api/metrics/steps/CoverageViewModel.java

```diff
@@ -375,7 +375,7 @@ public Object getDynamic(final String link, final StaplerRequest request, final
                 Optional<Node> targetResult
                         = getNode().findByHashCode(Metric.FILE, Integer.parseInt(link));
                 if (targetResult.isPresent() && targetResult.get() instanceof FileNode) {
-                    String tableId = request.getParameter(TABLE_ID);
+                    String tableId = request.getParameter(TABLE_ID) != null ? request.getParameter(TABLE_ID) : ABSOLUTE_COVERAGE_TABLE_ID;
```

Comment:
Prefer to change the code structure to prevent duplicate calls:
```
String tableId = request.getParameter(TABLE_ID);
if (tableId == null) {
    tableId = ABSOLUTE_COVERAGE_TABLE_ID;
}
```

Reply:
Btw, just to understand the logic, why do we need this here?